__docs__/wonder-blocks-dropdown/multi-select-variants.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from "react";
```

Comment:
Inspired by the [Icon Button Variants stories](https://github.com/Khan/wonder-blocks/blob/main/__docs__/wonder-blocks-icon-button/icon-button-variants.stories.tsx) :D